schema/deploy/type_statuses.sql

```diff
@@ -0,0 +1,8 @@
+-- Deploy ggircs-portal:type_statuses to pg
+-- requires: schema_ggircs_portal
+
+begin;
+
+create type ggircs_portal.statuses as enum ('pending', 'approved', 'rejected');
```

Comment:
The name of this type should be more precise. `user_organisation_status` is better, as it tells us in which context the type is used

Reply:
:+1: